src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -24,15 +24,20 @@
 require "y2storage/dialogs/guided_setup/base"
 
 Yast.import "InstExtensionImage"
+Yast.import "Popup"
 
 module Y2Storage
   module Dialogs
     class GuidedSetup
       # Dialog to select partitioning scheme.
       class SelectScheme < Base
-        def encryption_handler
-          widget_update(:password, widget_value(:encryption), attr: :Enabled)
-          widget_update(:repeat_password, widget_value(:encryption), attr: :Enabled)
+        # Handler for :encryption check box.
+        # @param allow_focus [Boolean] whether password field can be focused
+        def encryption_handler(allow_focus: true)
```

Comment:
well, `focusable` indicate that something is focusable. In libyui focusable == enabled. So it is confusing. You do not set something to be focusable, but to change focus under certain conditions.